docs-web/integration-with-cis.md

```diff
@@ -98,7 +98,7 @@ The name of the resource and the labels in the selector must match the values yo
 
 Now the Ingress Controller pods are behind the IP configured in Step 2.
 
-If you deploy the [cafe example](https://github.com/nginxinc/kubernetes-ingress/tree/master/examples/complete-example), you will be able to send requests to the Ingress Controller pods using the following command:
+If you deploy the [cafe example](https://github.com/nginxinc/kubernetes-ingress/tree/v.1.11.0/examples/complete-example), you will be able to send requests to the Ingress Controller pods using the following command:
```

Comment:
could you remove this file for now (and also update https://github.com/nginxinc/kubernetes-ingress/blob/b50c771b813a9120a27bf1b861d6028fd9f7f0e4/docs-web/index.rst )? we will restore it once we officially publish this integration

Reply:
fixed